xml/art_sle_ha_nfs_quick.xml

```diff
@@ -29,6 +29,17 @@
    <para>
     &abstract-nfsquick;
    </para>
+   <warning>
+    <title>This guide is no longer recommended</title>
+    <para>
+     The method described in this version of the guide is outdated and may cause issues in some
+     setups. For more information, see <link xlink:href="https://www.suse.com/support/kb/doc/?id=000020396"/>.
+    </para>
+    <para>
+     The process for configuring highly available NFS storage has been improved in
+     version 15 SP4.
```

Comment:
15 SP3 ?

Reply:
Technically yes, but I wondered if we didn't want to direct people to a version that's now in LTSS. But I suppose that will also be true of SP4 eventually. Now that we've finalised the doc strategy for 15 SP3, then yeah it does make sense to say it here.